### ISSUE TITLE
Node 16 actions are EOL and should be bumped to node 20

### ISSUE THREAD

**Issue by mattrasband** [mnavarrocarter/authorize-aws-security-group-ingress-action#64]
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: mnavarrocarter/authorize-aws-security-group-ingress-action@v1.0.0. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

Would it be possible to update this action to use node 20?

**Comment by mattrasband** [comment]
This would be fixed by #66 .

**Comment by mattrasband** [comment]
Closing this issue as it is on the radar with that pr :)
